src/larktools/evaluation.py

```diff
@@ -100,7 +100,7 @@ def eval_neg_atom(node, env):
     # character. Therefore, it doesn't appear among the child nodes
     child = get_children(node)[0]
     assert get_name(child) == "atom"
```

Comment:
Shouldn't this be:
```suggestion
    assert get_name(child) == "neg_atom"
```

Reply:
Tests are already passing with your version, so that seems to be the good version already. I guess any `child` of `neg_atom` is of type `atom`, so your version is already correct.